introduction.md

```diff
@@ -1,2 +1,5 @@
 My name is John Creson and I'm into test automation.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?